CHANGELOG.md

```diff
@@ -62,6 +62,13 @@
   performance.  
   [jpsim](https://github.com/jpsim)
 
+* Add `--stable-git-revision` option to the lint command. This allows
+  specifying a git revision or "commit-ish" that is considered stable.
+  If specified, SwiftLint will attempt to query the git repository index
+  for files changed since that revision, using only those files as input
+  as opposed to traversing the file system to collect lintable files.  
+  [jpsim](https://github.com/jpsim)
```

Comment:
Would it be valuable to default to `HEAD`?

Reply:
I’d rather not have a default at all because of the aforementioned correctness pitfalls.

Reply:
Makes sense! It's fairly simple as well to provide a default on the implementation level 👌